pages/linux/logrotate.md

```diff
@@ -0,0 +1,24 @@
+# logrotate
+
+> Rotates, compresses, and mails system logs.
+> More information: <https://manned.org/logrotate.8>.
+
+- Force logrotate to run outside of normal interval:
```

Comment:
This sounds a little confusing, when you don't know that it runs in intervals. How about:
```suggestion
- Trigger a run of  logrotate manually:
```